addon/chrome/content/preferences/preferences.xhtml

```diff
@@ -72,6 +79,7 @@
       <menuitem id="tm-import" label="&settings.import;" oncommand="importData();"/>
       <menuitem id="tm-export" label="&settings.export;" oncommand="exportData();"/>
       <menuitem id="syncPrefs" label="&settings.sync;" oncommand="toggleSyncPreference(); this.checked=!this.checked;" type="checkbox"/>
+      <menuitem id="instantApply" label="Instant Preference?" oncommand="Services.prefs.setBoolPref('extensions.tabmix.instantApply', !!this.getAttribute('checked'))" type="checkbox"/>
```

Comment:
Changing the preference from here take effect only the next time that the option window is open.
Is it possible to make it work immediately ?

Reply:
Yes, but I think better not since Prefwindow are not designed that way. You have to handle everything by yourself.

Reply:
And it is because we only accept preference on window load that it only take effect the next time option window open.